PYME/Analysis/tile_pyramid.py

```diff
@@ -278,28 +310,30 @@ def __init__(self, storage_directory, pyramid_tile_size=256, mdh=None,
         self.n_tiles_x = n_tiles_x
         self.n_tiles_y = n_tiles_y
         self.depth = depth
-        
+
         self.x0 = x0
         self.y0 = y0
         self.pixel_size=pixel_size
         # TODO - should we be re-assigning these on load, not just when we create a new pyramid?
         self._mdh['Pyramid.x0'] = x0
         self._mdh['Pyramid.y0'] = y0
         self._mdh['Pyramid.PixelSize'] = pixel_size
-        
+
         if not os.path.exists(self.base_dir):
             os.makedirs(self.base_dir)
-            
+
         #self._tilecache = TileCache()
-            
-        self._imgs = PZFTileIO(base_dir=self.base_dir, suff='img')
```

Comment:
replace `PZFTileIO()` calls with `backend(...)`

Leaves current behaviour as default, but lets you over-ride it